_layouts/default.html

```diff
@@ -7,6 +7,13 @@
 {% seo %}
 
     <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    {% if page.favicon %}
```

Comment:
I believe this can be simplified to `{% assign favicon = page.favicion | default: site.favicon %}{% if favicon...`

Reply:
Thanks for the tips! I'm still quite new at Liquid template yet :p